frontends/bsd/src/app_root.tsx

```diff
@@ -251,25 +251,28 @@ export function AppRoot({ card, hardware, logger }: AppRootProps): JSX.Element {
     }
   }, [setStatus]);
 
-  const unconfigureServer = useCallback(async () => {
-    try {
-      await config.setElection(undefined);
-      await refreshConfig();
-      await logger.log(LogEventId.ElectionUnconfigured, userRole, {
-        disposition: 'success',
-        message:
-          'User successfully unconfigured the machine to remove the current election and all current ballot data.',
-      });
-      history.replace('/');
-    } catch (error) {
-      console.log('failed unconfigureServer()', error); // eslint-disable-line no-console
-      await logger.log(LogEventId.ElectionUnconfigured, userRole, {
-        disposition: 'failure',
-        message: 'Error in unconfiguring the current election on the machine',
-        result: 'Election not changed.',
-      });
-    }
-  }, [history, refreshConfig, logger, userRole]);
+  const unconfigureServer = useCallback(
+    async (options: { ignoreBackupRequirement?: boolean } = {}) => {
```

Comment:
I recommend reviewing this section with [`?w=1`](https://github.com/votingworks/vxsuite/pull/2378/commits/1d19547c3822b84829266b9a5f1ec9d90cbbd049?w=1)